sympy/core/power.py

```diff
@@ -982,8 +982,8 @@ def as_base_exp(self):
         """
 
         b, e = self.args
-        if b.is_Rational and b.p == 1 and b.q != 1:
-            return Integer(b.q), -e
+        if b.is_Rational and b.p < b.q and b.p > 0:
+            return 1/b, -e
         return b, e
```

Comment:
This change makes `((S(2)/3)**x).as_base_exp() -> 3/2, -x` instead of `2/3, x`

Reply:
Why is that a good change?

Reply:
it's canonical

Reply:
That should probably be handled in the generator routine, however.

Reply:
You can see how it affects a few expressions in the commit in which this was added.

Reply:
OK, this should be moved to `powsimp` #23918

Reply:
No, the intent is already there but only applied to 1/Integer. This function is like `as_numer_denom` in that it does some processing. The docstring indicates that `base` and `exp` are literal -- like `numer` and `denom` or `fraction`.